frontend/src/Authentication/Login.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  Container,
+  Grid,
+  Link,
+  TextField,
+  Typography,
+  makeStyles
+} from '@material-ui/core';
+
+import FacebookIcon from '@material-ui/icons/Facebook';
+import GTranslateIcon from '@material-ui/icons/GTranslate';
+import { GTranslate } from '@material-ui/icons';
+import {ThemeProvider,createMuiTheme} from '@material-ui/core/styles'
+import {green} from '@material-ui/core/colors'
+import {colors } from '@material-ui/core';
+import TopBar from '../TopBar'
+
+const theme = createMuiTheme({
+    palette:{
+      primary:{
+        main: green[500]
+      },
+      secondary: {
+        main: colors.lightBlue[500]
+      },
+      text: {
+        primary: colors.blueGrey[900],
+        secondary: colors.blueGrey[600]
+      }
+    }
+  
+    }
+  )
+  
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.background.dark,
+    height: '100%',
+    paddingBottom: theme.spacing(3),
+    paddingTop: theme.spacing(3)
+  }
+}));
+
+const Login = () => {
+  const classes = useStyles();
+  
+  return (
+    <>
+    <TopBar/>
+    <ThemeProvider theme = {theme}> 
+      <Box
+        display="flex"
+        flexDirection="column"
+        height="100%"
+        justifyContent="center"
+        padding = '5%'
+      >
+        <Container maxWidth="sm">
+                <Box mb={3}>
+                  <Typography
+                    color="textPrimary"
+                    variant="h3"
+                  >
+                    Sign in
+                  </Typography>
+                  <Typography
+                    color="textSecondary"
+                    gutterBottom
+                    variant="body2"
+                  >
+                    Sign in on the internal platform
+                  </Typography>
+                </Box>
+                <Grid
+                  container
+                  spacing={3}
+                >
+                  <Grid
+                    item
+                    xs={12}
+                    md={6}
+                  >
+                    <Button
+                      color="primary"
+                      fullWidth
+                      startIcon={<FacebookIcon />}
+                      size="large"
+                      variant="contained"
+                    >
+                      Login with Facebook
+                    </Button>
+                  </Grid>
+                  <Grid
+                    item
+                    xs={12}
+                    md={6}
+                  >
+                    <Button
+                      fullWidth
+                      startIcon={<GTranslate/>}
+                      size="large"
+                      variant="contained"
+                    >
+                      Login with Google
+                    </Button>
```

Comment:
For now, let's remove the "Sign in with X" options from this page as we don't have that functionality available from the backend. We can add it back later if needed.